service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/SendOrderReminderService.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.fpl.service.cmo;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.HearingBooking;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.time.LocalDateTime.now;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SendOrderReminderService {
+
+    public List<HearingBooking> getPastHearingBookingsWithoutCMOs(CaseData caseData) {
+        return caseData.getAllNonCancelledHearings().stream()
+            .map(Element::getValue)
+            .filter(booking -> booking.getEndDate().isBefore(now()) && !booking.hasCMOAssociation())
+            .collect(Collectors.toList());
```

Comment:
Do you think we need to sort on the hearing bookings?

Reply:
Should hopefully be sorted but just incase hearings have edited times, not sure the collection will be in the right order, so can add yep!